microsoft-edge/extensions-chromium/whats-new/released-features.md

```diff
@@ -5,13 +5,23 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 07/19/2021
+ms.date: 06/19/2023
 ---
 # Released features for Microsoft Edge Add-ons
 
+<!-- todo: add Sidebar -->
+
 The following features or updates have been launched on [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/) for Microsoft Edge extensions.
 
 
+<!-- ====================================================================== -->
+## Sidebar
+
+<!-- todo: add Sidebar summary sentence, link -->
```

Comment:
@Bhuvana-Priya - todo: add Sidebar summary sentence into this new h2 section i added

Reply:
added draft content/link